lib/elixir/pages/Operators.md

```diff
@@ -64,9 +64,16 @@ true
 The reason we can compare different data types is pragmatism. Sorting algorithms don’t need to worry about different data types in order to sort. For reference, the overall sorting order is defined below:
 
 ```
-number < atom < reference < functions < port < pid < tuple < maps < list < bitstring
+number < atom < reference < functions < port < pid < tuple < map < list < bitstring
 ```
 
+When comparing two `number` (a `number` is either of type `integer` or `float`) of different types, a conversion to the type with lesser precision will always occur, unless the comparison operator used is either `===` or `!==`. A `float` will be considered more precise than an `integer`, unless the `float` is greater/lesser than +/-9007199254740992.0, at which point all the significant figures of the `float` are to the left of the decimal point. This behavior exists so that the comparison of large `number` remains transitive.
+
+The collection types are compared using the following rules:
+* `tuple` are compared by size then element by element.
```

Comment:
We can do same as above here, and just say "tuples are compared ..." without backticks (which would mean dropping the trailing period as well so that we don't have to use uppercase initials).